workflow.json

```diff
@@ -1756,7 +1759,7 @@
                             "servicebus_1": {
                                 "connectionId": "[resourceId('Microsoft.Web/connections', parameters('sb_con_name'))]",
                                 "connectionName": "servicebus-connection",
-                                "id": "[subscriptionResourceId('Microsoft.Web/locations/managedApis', parameters('location'), 'servicebus')]"
```

Comment:
why don't you just pass location as uksouth or convert it before passing as a parameter?